develop/dev-guide-sample-application-java.md

```diff
@@ -637,11 +575,11 @@ The catalog structure is shown below：
 </hibernate-configuration>
 ```
 
-`HibernateExample.java` is the body of the `plain-java-hibernate` sample application. When we use Hibernate, compared to JDBC, we only need to write the configuration file address here, and Hibernate helps us to shield the details of the differences between databases when creating database connections.
+`HibernateExample.java` is the main body of the `plain-java-hibernate`. Compared with JDBC, when using Hibernate, it is only needed to write the path of configuration, as Hibernate avoid detailed difference between different databases when creating a database.
```

Comment:
```suggestion
`HibernateExample.java` is the main body of the `plain-java-hibernate`. Compared with JDBC, when using Hibernate, you only need to write the path of the configuration file, because Hibernate avoids differences in database creation between different databases.
```